components/Hero/index.js

```diff
@@ -41,17 +41,31 @@ class Hero extends Component {
 
     var backgroundColor = nameToRgba(color) || nameToRgba(Hero.defaultProps.color);
     var contentStyle = childrenPosition === 'after' ? {bottom: 'auto', top: '10em'} : EMPTY_OBJECT;
+    var innerCap = title;
+
+    if(this.props.title == 'Skookum'){
+      innerCap = React.DOM.img({src:'/public/images/wordmark.svg', alt:title});
+    }
+
+    if(this.props.title == 'Contact Us' || this.props.title == 'Innovation Camp'){
+      style.height = 90;
```

Comment:
We need the wordmark svg on the homepage where the title is. Passing that in as the title escapes the code. The other stuff is just a conditional for the smaller height hero on those 2 pages. Any suggestions on how to better handle?